lib/ui/text.dart

```diff
@@ -1565,14 +1565,46 @@ enum TextDecorationStyle {
   wavy
 }
 
+/// {@macro dart.ui.leadingDistribution}
+enum LeadingDistribution {
```

Comment:
Optional, but it seems that most text related enums/classes are prefixed with `Text`.

I don't have a particularly strong conviction either way, but It would also be reasonable to call this `TextLeadingDistribution` for consistency. That being said, as far as I know, the concept of leading is unique to text layout, so this naming should be sufficient (currently) to identify the feature uniquely.